pages/blog-post.tsx

```diff
@@ -0,0 +1,9 @@
+import BlogPost from "@components/BlogPost";
+
+const blog = () => {
+    return ( 
+        <BlogPost/>
+     );
+};
+
+export default blog;
```

Comment:
@oscarcorellan The name of the component and the name of the file should be on PascalCase
```suggestion
export default Blog;
```